bookkeeper-server/src/main/java/org/apache/bookkeeper/client/LedgerDeleteOp.java

```diff
@@ -94,4 +97,42 @@ public void safeOperationComplete(int rc, Void result) {
     public String toString() {
         return String.format("LedgerDeleteOp(%d)", ledgerId);
     }
+
+    public static class DeleteBuilderImpl  implements DeleteBuilder {
+
+        private long builderLedgerId = -1;
```

Comment:
for consistency sake make ledgerId Long or long in all the builders

Reply:
in this case we do not need a Long, it is enough a 'long'. The same is for OpenBuilder